client/src/components/Header.js

```diff
@@ -1,12 +1,34 @@
 import React from "react";
+import { connect } from "react-redux";
 import { Link } from "react-router-dom";
 import Profile from "./profile/Profile";
 import GoogleAuth from "./GoogleAuth";
 
 import "./style.css";
 import Logo from "./logonobg.png";
 
-const Header = () => {
+class Header extends React.Component {
+  renderCreate() {
+    console.log(this.props);
```

Comment:
Looks like you left this in. Other than that, looks good!

Reply:
Removed